src/docfx/Errors.cs

```diff
@@ -404,7 +404,7 @@ public static Error UidConflict(string uid, IEnumerable<string> conflicts = null
             }
```

Comment:
there is another case in line 403 where multiple UID is defined in the same file

Reply:
I didn't find the new message for the case in line 403 and the new message is not fit for the case in line 403 as it mentioned `more than one file`. Let me know if It needs update.

Reply:
It needs update, but probably not in this PR, @mingwli this is another item to confirm